test/mocks/ReconfigurationPayload.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import { IExecutorBase } from '../../src/interfaces/IExecutorBase.sol';
+
+/**
+ * @dev This payload reconfigures birdge executor to a given state
+ */
+contract ReconfigurationPayload {
+
+    function execute() external {
+        IExecutorBase(address(this)).updateDelay(getNewDelay());
+        IExecutorBase(address(this)).updateGracePeriod(getNewGracePeriod());
+        IExecutorBase(address(this)).updateMinimumDelay(getNewMinimumDelay());
+        IExecutorBase(address(this)).updateMaximumDelay(getNewMaximumDelay());
+        IExecutorBase(address(this)).updateGuardian(getNewGuardian());
+    }
+
+    function getNewDelay() public pure returns (uint256) {
+        return 1200;
+    }
+
+    function getNewGracePeriod() public pure returns (uint256) {
+        return 1800;
+    }
+
+    function getNewMinimumDelay() public pure returns (uint256) {
+        return 100;
+    }
+
+    function getNewMaximumDelay() public pure returns (uint256) {
+        return 3600;
+    }
+
+    function getNewGuardian() public pure returns (address) {
+        return 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
```

Comment:
Would pass in a `makeAddr("newGuardian")` in the constructor.

Reply:
I can't really do that. It's gonna be in the payload storage and the delegate call from executor is gonna run on the executor's storage.

Reply:
For now changed it to something more obviously meaningless.

Reply:
Ok, immutable is the thing 😅 Fixed.